lib/kitchen/directions/bake_chapter_references/v1.rb

```diff
@@ -4,11 +4,12 @@ module Kitchen::Directions::BakeChapterReferences
   class V1
     renderable
 
-    def bake(chapter:, metadata_source:, uuid_prefix: '.', klass: 'references')
+    def bake(chapter:, metadata_source:, uuid_prefix: '.', klass: 'references', from_introduction: false)
       @metadata = metadata_source.children_to_keep.copy
       @klass = klass
       @title = I18n.t(:references)
       @uuid_prefix = uuid_prefix
+      @from_introduction = from_introduction
```

Comment:
if this variable is not used in the template (and it doesn't look like it is), this line isn't necessary!